app/controllers/admin/induction_coordinators_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Admin
+  class InductionCoordinatorsController < Admin::BaseController
+    skip_after_action :verify_authorized, only: :index
+    skip_after_action :verify_policy_scoped, except: :index
+    before_action :load_induction_coordinator, only: %i[edit update]
+
+    def index
+      @induction_coordinators = policy_scope(User).induction_coordinators
+    end
+
+    def edit
+      authorize User
```

Comment:
Should it be `authorize @ induction_coordinator`? Here and in update?

Reply:
rebase went wrong, will fix